app/views/rides/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for [@ride] do |f|%>
+<%= f.label :name, "Ride Name" %>
+<%= f.text_field :name %>
+
+<%= f.label :location, "Ride Location" %>
+<%= f.select :location, ['Frontierland', 'Fantasyland', 'Liberty Square', 'Tomorrowland', 'Adventureland', 'Main Street, U.S.A' ] %>
+
+<%= f.label :image_url %>
+<%= f.url_field :image_url %>
+
+<%= f.label :description, "Ride Description" %>
+<%= f.text_area :description, size: "70x3" %>
+
+<%= f.submit %>
```

Comment:
these form helpers should be indented in 

on line 1, prefer removing those brackets